src/code/z_sram_NES.c

```diff
@@ -35,70 +35,90 @@ PersistentCycleSceneFlags sPersistentCycleSceneFlags[SCENE_MAX] = {
 #undef DEFINE_SCENE
 #undef DEFINE_SCENE_UNSET
 
-#define WEEK_FLG2(flg) (2 << (2 * BIT_FLAG_TO_SHIFT(flg)))
-#define WEEK_FLG(flg) (3 << (2 * BIT_FLAG_TO_SHIFT(flg)))
+// Each flag has 2 bits to store persistence over the three-day reset cycle
+// Only 1 of these bits need to be set to persist (Values 1, 2, 3).
+// Therefore, these two macros are functionally identical
+#define PERSISTENT_WEEKEVENTREG(flag) (3 << (2 * BIT_FLAG_TO_SHIFT(flag)))
+#define PERSISTENT_WEEKEVENTREG_ALT(flag) (2 << (2 * BIT_FLAG_TO_SHIFT(flag)))
 
 // weekEventReg flags which will be not be cleared on a cycle reset
+//! @note The index of this array must manually be aligned with the index of the WeekeventReg
```

Comment:
I don't think "aligned" is a good word to use here since it has other meaning, maybe more like
```suggestion
//! @note The index of the flag in this array must be the same to its index in the WeekeventReg array
```
?